base/base.bzl

```diff
@@ -103,6 +103,7 @@ def distro_components(distro):
                 workdir = workdir,
                 os = "linux",
                 architecture = arch,
+                variant = VARIANTS.get(arch),
```

Comment:
presumably this just bubbles down to all images?

Reply:
Yes